var.tfvars

```diff
@@ -30,7 +30,7 @@ pull_secret_file            = "data/pull-secret.txt"
 cluster_domain              = "example.com"
 cluster_id_prefix           = "test"
 
-dns_forwarders              = "8.8.8.8; 8.8.4.4"
+dns_forwarders              = "1.1.1.1; 9.9.9.9"
```

Comment:
Is the 9.9.9.9 public ?

Reply:
@bpradipt 
yep, it is https://www.quad9.net/

Reply:
Thanks @rpsene